vpr/src/place/place_util.h

```diff
@@ -14,7 +14,11 @@
 #include "vtr_vector_map.h"
 #include "globals.h"
 
-
+/**
+ * @brief The error tolerance due to round off for the total cost computation.
+ * When we check it from scratch vs. incrementally. 0.01 means that there is a 1% error tolerance.
+ */
+constexpr double ERROR_TOL = .01;
```

Comment:
ERROR_TOL is a very common variable name to be putting into the global scope. Could you put this in a `place` namespace or into some class. This is very likely to conflict.

Reply:
I think creating a new namespace for this constant might be overkill. What do you think about renaming it to `PLACEMENT_INCREMENTAL_COST_ERROR_TOL` instead?

Reply:
Good for me! Its only used once or twice and only for that increment cost stuff. I just wanted to avoid ERROR_TOL since even I use that variable name sometimes.

Reply:
Fine with me, but I'd probably use PL_INCREMENTAL_COST_TOLERANCE to make it a bit shorter but still distinct.